### ISSUE TITLE
How do changes to the rules/structure get approved? 

### ISSUE THREAD

**Issue by brettneese** [HackathonHackers/org#14]
Who is in charge of the repos, and under what conditions does a pull request get merged?

**Comment by hallie** [comment]
I agree with what @rubinovitz said about how making the moderation team basically a group of elected janitors will probably discourage a number of really good people from wanting to be a part of it. I think that maybe the elected mods can also serve on a team that handles the Code of Conduct (as an automatic) and then have a subset of them tasked with managing the GitHub org in general (creating new teams, and making sure PRs are merged for org and such).

But there should also be other teams that just work on specific things. Like how we used to have a Data team. But then that calls into question how we decide who leads those teams/decides who gets to be on them.

**Comment by rubinovitz** [comment]
The data team worked well with being contribution based.

**Comment by rubinovitz** [comment]
I think we need a board to lead policy decisions and direction but those could even be voted on by members.

**Comment by rubinovitz** [comment]
I think that's why electing a TEAM with ownership is important because they have a lot of work to do as far as processes.

**Comment by brettneese** [comment]
Was thinking about this in my sleep last night. Here's my thought, I can formalize it later:

I think it's really important that the elected mods/"janitors" aren't the ones making + enforcing the rules. 

With Hackbot, we already have the ability to have granular controls/multiple levels of mods. May make sense to split up into a legislative and executive branch that check/balance each other. The executive branch ("mods") would  be max 3 people, and would have access to ownership of properties, etc, and would be on the Facebook group's admin functionality; the legislative branch ("board") is tasked with considering and approving new rules/the code of conduct/bylaws, etc. 

Those ~3 people then get to choose a team of people under them to act as hackbot janitors. They are tasked merely with enforcing the CoC. 

The board can call into question decisions to ie, remove posts, and elect to remove a hackbot janitor but only with a extremely high barrier and due process. Subgroup leaders can elect to do a vote of no confidence for the whole team, which will call another election. Hackathon Hackers in general can petition for impeachment of specific members of the board or mod (not sure about this, not clear how much power we should distribute to HH; we do not want "majority rule"/factions to control.) The general membership can, of course, suggest, petition, lobby, etc the legislative branch for new rules/change of policy/etc, and is welcome to submit issues and PRs, but it's up to the legislative branch to approve them after meeting and ensuring that they're maximally fair.

Obviously there's a lot of trust to consider and distribute and ie, we don't have a way to audit the Facebook admin team, which is why it should be small (but larger than 1 person). You're not going to create a fraud-free system, but you can try and eliminate tyranny, even in a system with a ultimate superadmin, and someone needs to be in that position to control the handoff of control.

**Comment by hallie** [comment]
@brettneese That sounds really interesting. How would the super admins be decided?

**Comment by brettneese** [comment]
Not sure. Right now it has to be @hellyeah because Facebook but lobbying
them to let us change it seems like a possibility in the future. We should
think about how that would work.

**Comment by hallie** [comment]
@brettneese O, wait, I meant the admin panel :P

**Comment by rubinovitz** [comment]
Hi all,

Had intended to post more thoughts on this today, but my cousin is on life
support and I need to deal with that. Hoping to be back in 48 hours.

**Comment by hallie** [comment]
@rubinovitz All good. Take all the time you needs

**Comment by brettneese** [comment]
@hallie not sure yet. Still thinking about this. Probably should come from
the main group, with maybe some kind of restriction on how long they've
been in group to prevent rigging.

Could do opposite as well; where board (Facebook admins) gets pulled from
subgroup leaders and mod team (hack or admins) gets pulled from General.

The idea is you want to balance where power comes from so that it works in
interest of everyone. That's the way a lot of NPOs work too, where one team
(usually employees) executes on vision and rules set by board (usually not
employees), not to mention most democratic governments (🇺🇸🇺🇸🇺🇸🇺🇸).

Would love to hear others' thoughts. Will post in HH main. Biggest concern
is eliminating factions and tyranny. What the majority might want may not
actually be in their best interest or in the interest of minorities.

@jb no worries, sending my thoughts your way. 😱

On Saturday, January 2, 2016, rubinovitz notifications@github.com wrote:

> Hi all,
> 
> Had intended to post more thoughts on this today, but my cousin is on life
> support and I need to deal with that. Hoping to be back in 48 hours.
> 
> On Sat, Jan 2, 2016, 10:44 PM Brett Neese <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
> 
> > Not sure. Right now it has to be @hellyeah because Facebook but lobbying
> > them to let us change it seems like a possibility in the future. We
> > should
> > think about how that would work.
> > 
> > On Saturday, January 2, 2016, Hallie Lomax <notifications@github.com
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>>
> > wrote:
> > 
> > > @brettneese https://github.com/brettneese That sounds really
> > > interesting. How would the super admins be decided?
> > > 
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> > > https://github.com/HackathonHackers/org/issues/14#issuecomment-168456960
> > > 
> > > .
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > <
> > https://github.com/HackathonHackers/org/issues/14#issuecomment-168457007>
> > .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/HackathonHackers/org/issues/14#issuecomment-168457090
> .

## 

Brett Neese
563-210-3459
